student.rb

```diff
@@ -9,6 +10,11 @@ def initialize(classroom, age, name = 'Unknown', parent_permission: true)
     @parent_permission = parent_permission
   end
 
+  def owner=(classroom)
+    @classroom = classroom
+    classroom.students.push(self) unless classroom.students.include?(self)
+  end
+
```

Comment:
- [ ] in order to make this work you need to add the attr_reader for `classroom`